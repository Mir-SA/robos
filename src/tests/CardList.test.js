import React from "react";
import { shallow } from "enzyme";
import CardList from "../components/CardList";

it("expect to render CardList component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "Amitabh Bacchan",
      username: "BigB",
      email: "Bigb@yahoo.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
