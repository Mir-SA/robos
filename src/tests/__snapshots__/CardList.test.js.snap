// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <Card
    email="Bigb@yahoo.com"
    id={1}
    key="0"
    name="Amitabh Bacchan"
  />
</div>
`;
